test/classes/eik-config/mappings.test.js

```diff
@@ -302,7 +305,7 @@ test('mappings - files is an object - mapped to folder - absolute path to folder
         {
             ...validEikConfig,
             files: {
-                folder: join(baseDir, 'folder'),
+                folder: join(baseDir, 'folder').replace(/\\/g, '/'),
```

Comment:
Looking at the number of places I do this, this replace ought to be a util function